plugins/scaffolder-react/src/next/components/Stepper/createAsyncValidators.ts

```diff
@@ -15,27 +15,41 @@
  */
 
 import { FieldValidation } from '@rjsf/utils';
-import { JsonObject } from '@backstage/types';
+import type { JsonObject, JsonValue } from '@backstage/types';
 import { ApiHolder } from '@backstage/core-plugin-api';
 import { Draft07 as JSONSchema } from 'json-schema-library';
 import { createFieldValidation } from '../../lib';
 import { NextCustomFieldValidator } from '../../extensions';
 
+function isObject(value: JsonValue | undefined): value is JsonObject {
```

Comment:
Think this should also live in the `utils` file too right?

Reply:
yes, that makes sense.